ansible/roles/cloud-node/tasks/main.yml

```diff
@@ -87,7 +87,7 @@
     - name: Packages                   | Add Percona repository
       when: ansible_os_family == 'RedHat'
       yum:
-        name: https://www.percona.com/redir/downloads/percona-release/redhat/0.1-4/percona-release-0.1-4.noarch.rpm
+        name: https://repo.percona.com/yum/percona-release-0.1-7.noarch.rpm
```

Comment:
As I know 1.0-6 package already available So please check if it is possible to use it

Reply:
thanks testing :+1: